book/src/developers/contributing/rust/style.md

```diff
@@ -0,0 +1,10 @@
+# Style
```

Comment:
There have been some [updates](https://github.com/ethereum/trin/commit/b91e976b7f3603ed310859005c6fd2810e4c7c6d) to the style guide, so just make sure to rebase and keep these new documents up to date with the latest master as this pr goes through the review process.

Reply:
Thanks, yes I'll keep an eye out